node/node.go

```diff
@@ -151,7 +151,7 @@ func New(cfg *Config, version string) (*Node, error) { //nolint:gocyclo,funlen
 			return nil, kErr
 		}
 
-		sequencer := builder.New(pKey, new(felt.Felt).SetUint64(1337), chain, nodeVM, log) //nolint: gomnd
+		sequencer := builder.New(pKey, new(felt.Felt).SetUint64(1337), chain, nodeVM, time.Minute, log) //nolint: gomnd
```

Comment:
I think this should be (eg) `time.Now().Unix()`, since `time.Minute` always outputs "1m0s"